src/main/java/org/coursesjava/repository/AccountRepositoryImpl.java

```diff
@@ -0,0 +1,80 @@
+package org.coursesjava.repository;
+
+import org.coursesjava.model.Account;
+import org.coursesjava.model.User;
+import org.coursesjava.repository.dao.AccountRepository;
+
+import java.sql.*;
+
+public class AccountRepositoryImpl implements AccountRepository {
+    private final Connection connection;
+
+    public AccountRepositoryImpl(Connection connection) {
+        this.connection = connection;
+    }
+
+    private final String create =
```

Comment:
it would be better to move each statement to the appropriate method, also final modificator is redundant